test/disclosure_permission_bloc_test.dart

```diff
@@ -1995,6 +1995,112 @@ void main() {
 
     await repo.getSessionState(45).firstWhere((session) => session.status == SessionStatus.success);
   });
+
+  test('credential-status-sorting', () async {
+    // Issue revoked credential
+    await _issueCredential(
+      repo,
+      mockBridge,
+      43,
```

Comment:
Minor thing, maybe it's nice to begin with 42 and give the second issuance session 43. Then the session order is nicely visible in the session IDs.